crates/air-lib/air-parser/src/ast/instruction_arguments.rs

```diff
@@ -25,7 +25,7 @@ use air_lambda_ast::LambdaAST;
 use serde::Deserialize;
 use serde::Serialize;
 
-#[derive(Debug, PartialEq, Clone, Serialize, Deserialize)]
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
```

Comment:
Is adding `Eq` solves some warning, or "just in case"?

Reply:
Yep, it solves `clippy` warning.